src/test/java/core/basesyntax/strategy/impl/BalanceOperationStrategyTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationStrategyTest {
+    private static BalanceOperationStrategy balanceOperationStrategy;

```

Comment:
```suggestion
    private static OperationStrategy balanceOperationStrategy;
```
the same for other implementations test classes